pkg/config/application.go

```diff
@@ -416,6 +417,11 @@ func (c *CustomSyncOptions) Validate() error {
 	return nil
 }
 
+type ExternalTool struct {
+	Package string `json:"package"`
+	Version string `json:"version"`
+}
```

Comment:
we should have validation for this type, since there is no mean for setting external tools without `Package` value, also how do we handle the case version is not provided? 🤔

Reply:
> how do we handle the case version is not provided
I think setting latest version is best.